drivers/linstorvolumemanager.py

```diff
@@ -1741,6 +1747,21 @@ def get_resources_info(self):
 
         return resources
 
+    def get_resource_info(self, volume_uuid: str) -> Dict:
+        """
+        Give all resources info with provided uuid on current group name.
+        :param: volume_uuid
+        :rtype: dict
+        """
+        for volume in self.get_resources_info().values():
```

Comment:
The cost can be really important. I guess we can extract a few logic of `get_resources_info` to reduce this expense.